src/Profile/InfoBoxConfig.cpp

```diff
@@ -123,6 +123,10 @@ Profile::Load(const ProfileMap &map, InfoBoxSettings &settings)
     break;
   }
 
+  map.Get(ProfileKeys::InfoBoxScreenRatio, settings.ib_to_screen_ratio);
+  if (settings.ib_to_screen_ratio < 5.99 || settings.ib_to_screen_ratio > 10.01)
+    settings.ib_to_screen_ratio = 7.4;
```

Comment:
Why set this arbitrary value if the configured value is out of range? Why not clamp?

Reply:
7.4 was the original value (hard coded in InfoBoxes/InfoBoxLayout.cpp).
The addition here is meant to initialize the value to what the user had seen before.

Reply:
That should sure be the default value until the user has configured something else. But that's not what your code expresses to do. It does that, but in a very obscure way.

Reply:
OK.